.scalafmt.conf

```diff
@@ -47,8 +75,40 @@ rewrite.neverInfix.excludeFilters = [
   theSameElementsAs
   theSameElementsInOrderAs
 ]
-rewriteTokens = {
+rewriteTokens          = {
   "⇒": "=>"
   "→": "->"
   "←": "<-"
 }
+project.excludeFilters = [
+  "scripts/authors.scala"
+]
+fileOverride           = {
+  "glob:**/akka-actor-typed/src/main/scala-2.12/**" {
+    runner.dialect = scala212
+  }
+  "glob:**/akka-actor-typed/src/main/scala-3/**" {
+    runner.dialect = scala3
+  }
+  "glob:**/akka-cluster-typed/src/main/scala-2.12/**" {
+    runner.dialect = scala212
+  }
+  "glob:**/akka-cluster-typed/src/main/scala-3/**" {
+    runner.dialect = scala3
+  }
+  "glob:**/akka-distributed-data/src/main/scala-3/**" {
+    runner.dialect = scala3
+  }
+  "glob:**/akka-actor-tests/src/test/scala-2.12/**" {
+    runner.dialect = scala212
+  }
+  "glob:**/akka-actor-tests/src/test/scala-3/**" {
+    runner.dialect = scala3
+  }
+  "glob:**/akka-actor/src/main/scala-2.12/**" {
+    runner.dialect = scala212
+  }
+  "glob:**/akka-actor/src/main/scala-3/**" {
+    runner.dialect = scala3
```

Comment:
Definitely true, I just wanted to be a bit more strict incase I ended up grabbing stuff that I shouldn't have been (such as generated source files or something along those lines). Initially I want to be a safer side however I have no problem with the `.scalafmt.conf` being changed later if it doesn't updating anything.

Reply:
1. generated files won't be touched since you have git enabled
2. all these patterns can be replaced with https://scalameta.org/scalafmt/docs/configuration.html#projectlayout

Reply:
I just rebased the PR using `project.layout = StandardConvention` and its working as expected. thanks for the tip!